includes/payment-methods/class-wc-stripe-upe-payment-method.php

```
@@ -197,6 +197,17 @@ public function is_enabled() {
 	 * @return bool
 	 */
 	public function is_available() {
+		if ( is_add_payment_method_page() ) {
+			if ( ! $this->is_reusable() ) {
+				return false;
+			}
+
+			$main_stripe_gateway = WC_Stripe::get_instance()->get_main_stripe_gateway();
```

PHPMD (2.15.0)

[error] The variable $main_stripe_gateway is not named in camelCase. (undefined)

(CamelCaseVariableName)